scraper.php

```diff
@@ -0,0 +1,98 @@
+    curl_setopt($curl, CURLOPT_HEADER, TRUE);
+    $terms_response = curl_exec($curl);
+    curl_close($curl);
+    // get cookie
+    // Please imporve it, I am not regex expert, this code changed ASP.NET_SessionId cookie
+    // to ASP_NET_SessionId and Path, HttpOnly are missing etc
+    // Example Source - Cookie: ASP.NET_SessionId=bz3jprrptbflxgzwes3mtse4; path=/; HttpOnly
+    // Stored in array - ASP_NET_SessionId => bz3jprrptbflxgzwes3mtse4
+    preg_match_all('/^Set-Cookie:\s*([^;]*)/mi', $terms_response, $matches);
+    $cookies = array();
+    foreach($matches[1] as $item) {
+        parse_str($item, $cookie);
+        $cookies = array_merge($cookies, $cookie);
+    }
+    return $cookies;
+}
+
+###
+### Main code start here
+###
+$url_base = "https://online.kingston.vic.gov.au/ePathway/Production/Web/GeneralEnquiry/";
+$term_url = "https://online.kingston.vic.gov.au/ePathway/Production/Web/GeneralEnquiry/EnquiryLists.aspx";
+$user_agent = "User-Agent:Mozilla/5.0 (Windows NT 6.1; WOW64) PlanningAlerts.org.au";
+
+$da_page = $url_base . "EnquirySummaryView.aspx?SortFieldNumber=5&SortDirection=Descending";
+$comment_base = "mailto:info@kingston.vic.gov.au?subject=Development Application - ";
+
+$cookies = accept_terms_get_cookies($term_url, "Next", array('mDataGrid:Column0:Property' => 'ctl00$MainBodyContent$mDataList$ctl01$mDataGrid$ctl02$ctl00'));
+
+# Manually set cookie's key and get the value from array
+$request = array(
+    'http'    => array(
+    'header'  => "Cookie: ASP.NET_SessionId=" .$cookies['ASP_NET_SessionId']. "; path=/; HttpOnly\r\n".
+                 "$user_agent\r\n"
+    ));
+$context = stream_context_create($request);
+$dom = file_get_html($da_page, false, $context);
+
+# Assume it is single page, the web site doesn't allow to select period like last month
+$dataset  = $dom->find("tr[class=ContentPanel], tr[class=AlternateContentPanel]");
+
+# The usual, look for the data set and if needed, save it
+foreach ($dataset as $record) {
+    # Slow way to transform the date but it works
+    $date_received = trim(html_entity_decode($record->find('td',5)->plaintext));
+    $date_received = explode('/', $date_received);
+    $date_received = "$date_received[2]-$date_received[1]-$date_received[0]";
+    
+    $address = trim(html_entity_decode($record->find('td', 1)->plaintext)) . ', ' .
+               trim(html_entity_decode($record->find('td', 2)->plaintext)) . ', ' . 'VIC';
+    $address = preg_replace('/\s+/', ' ', $address);
+
+    # Put all information in an array
+    $application = array (
+        'council_reference' => trim(html_entity_decode($record->find('a',0)->plaintext)),
+        'address'           => $address,
+        'description'       => preg_replace('/\s+/', ' ', trim(html_entity_decode($record->find('td', 3)->plaintext))),
+        'info_url'          => $term_url,
+        'comment_url'       => $comment_base . trim(html_entity_decode($record->find('a',0)->plaintext)),
```

Comment:
Because this isn't URL encoded I don't think it will validate. This is something that PlanningAlerts checks so I'd suggest just setting it to the base `mailto:` address without a pre-baked subject.